exasol_advanced_analytics_framework/udf_communication/peer_communicator/background_listener_thread.py

```diff
@@ -49,22 +53,29 @@
 def create_background_peer_state_builder() -> BackgroundPeerStateBuilder:
     timer_factory = TimerFactory()
     abort_timeout_sender_factory = AbortTimeoutSenderFactory()
+    peer_is_ready_sender_factory = ConnectionIsReadySenderFactory()
```

Comment:
Naming is a bit inconsistent here, peer-is-ready-sender-factory or connection-is-ready-sender-factory?

Reply:
yeah, this is probably a leftover from the before the refactoring, will fix it